### DISCUSSION TITLE
Error after upgrading to Bonsai 2.8.5: the ' ' character, hexadecimal value 0x20, cannot be included in a name

### DISCUSSION THREAD

**Discussion by edmundchong** [bonsai-rx/bonsai#1973] [Q&A] [answered]
Hello! After recently upgrading to 2.8.5, the same bonsai workflows that were working before, started throwing this error 
`The ' ' character, hexadecimal value 0x20, cannot be included in a name`

I have a main workflow (let's call it Main), and I have isolated the error to two separate SavedWorkflows that are nested within Main with `IncludeWorkflow` (let's call them A, and B). 

1. Trying to save Main throws this error.
2. If I ungroup A and B (right click, and select Ungroup), I can save Main without throwing the error.
3. I can ungroup A and B, and still `SaveAsWorkflow` for each of them
4. Trying to copy A or B (clicking on either of them and pressing Ctrl-C), throws this error
5. I use a whole bunch of other SavedWorkflows in Main, but they do not throw this error.

**Comment by glopesdev** [comment]
@edmundchong thanks for reporting, can you attach the example workflows to see if we can reproduce this?

**Reply by edmundchong** [reply, marked as answer]
@glopesdev Attached is the workflow folder. main.bonsai contains the IncludeWorkflow that calls "StageChecker.bonsai". This has no errors in bonsai 2.7.2, but in 2.8.5 I get the characteristics listed in the original post :)

[StageChecker.zip](https://github.com/user-attachments/files/16709084/StageChecker.zip)

Thanks again!

**Reply by banchan86** [reply]
bug report has been filed @ https://github.com/bonsai-rx/bonsai/issues/2398, thanks @edmundchong for noticing this issue!
